website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -73,6 +73,47 @@ resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
 }
 ```
 
+## Example Usage implementing at least one Policy Item
```

Comment:
So this is taking me a while to review and after testing it all out I think we need 3 examples:

## Example Usage - Create a Cluster and Modify the 4 Default Policies Simultaneously
_This is the existing example we already had but just need to change the header_

## Example Usage - Create a Cluster and Modify 3 Default Policies and Remove 1 Default Policy Simultaneously
_This needs to be a new example, similar to the existing example that just removes one of the default policy blocks_

## Example Usage - Remove 3 Default Policies Items After the Cluster Has Already Been Created
_Same example that's already here._  
 
Each example also needs a note to explain what it is doing:
First example:
-> **NOTE:** When Cloud Backup is enabled for a cluster MongoDB Atlas automatically creates a default Cloud Backup schedule for the cluster with four policy items; hourly, daily, weekly, and monthly.   Because of this default creation this provider automatically saves the Cloud Backup Snapshot Policy into the Terraform state.  If the default works well for you then you do not need to do anything other than create a cluster with Cloud Backup enabled and your Terraform state will have this information if you need it. However,  if you want the policy to be different than the default when creating the cluster simply follow this example to modify the four default policies.

Second example:
-> **NOTE:** (See text in first example for more details on the default.) If you want the Cloud Backup Snapshot Policy to vary in the number of policies from the default when creating the cluster, perhaps you want to remove one policy item and modify the remaining three, simply follow this example here to remove a policy and modify three.

Third example:
-> **NOTE:** (See text in first example for more details on the default.) If you want the Cloud Backup Snapshot Policy to vary in number of policies for a cluster that was already created/imported, perhaps you want to remove three policy items and modify the remaining policy, simply follow the example here.

Reply:
I like it it's a great idea. I will implement it, thanks. 👍